### DISCUSSION TITLE
初始配置失败

### DISCUSSION THREAD

**Discussion by henrywang1201** [MoeNetwork/Tieba-Cloud-Sign#269] [疑问]
在web ui进行初始配置时失败，显示`php_network_getaddresses: getaddrinfo for db failed: Name has no usable address`
安装系统是unraid，安装时没有使用docker compose而是直接配置docker，是不是没有连接上mariadb数据库

**Comment by n0099** [comment]
https://github.com/MoeNetwork/Tieba-Cloud-Sign/discussions/257
